src/Elastic.Apm/DiagnosticListeners/HttpDiagnosticListenerImplBase.cs

```diff
@@ -153,10 +153,6 @@ private void ProcessStartEvent(object eventValue, TRequest request, Uri requestU
 					Url = requestUrl.ToString(),
 					Method = RequestGetMethod(request)
 				};
-
-				var frames = new StackTrace(true).GetFrames();
```

Comment:
Where did this go? We captured stacktraces for outgoing web requests on purpose, this is also shown on the UI. I think the plan was to add a debug flag and not do it by default.

Reply:
Please see my response in the other comment on this subject.